packages/jbr/lib/task/ExperimentLoader.ts

```diff
@@ -23,8 +25,11 @@ export class ExperimentLoader {
   public static readonly COMBINATIONS_NAME = 'jbr-combinations.json';
   public static readonly PACKAGEJSON_NAME = 'package.json';
   public static readonly PREPAREDMARKER_PATH = [ 'generated', '.prepared' ];
-  public static readonly IRI_EXPERIMENT_HANDLER = `https://linkedsoftwaredependencies.org/bundles/npm/jbr/^${pJson.version}/components/experiment/ExperimentHandler.jsonld#ExperimentHandler`;
-  public static readonly IRI_HOOK_HANDLER = `https://linkedsoftwaredependencies.org/bundles/npm/jbr/^${pJson.version}/components/hook/HookHandler.jsonld#HookHandler`;
+  public static readonly IRI_EXPERIMENT_HANDLER = `https://linkedsoftwaredependencies.org/bundles/npm/jbr/` +
+  `^${coerce(major(pJson.version))!.raw}/components/experiment/ExperimentHandler.jsonld#ExperimentHandler`;
```

Comment:
Can this not be simplified? E.g. https://github.com/LinkedSoftwareDependencies/Components-Generator.js/blob/69eefcef6606f3d256571edd2f6ff58232749e93/lib/parse/PackageMetadataLoader.ts#L37

Reply:
Yep added it